### ISSUE TITLE
Split image stack to seperate bands

### ISSUE THREAD

**Issue by tyler-c2s** [cloudtostreet/Sen1Floods11#4]
In order to improve cloud storage access and ML training we should split the sentinel-2 and sentinel-1 imagery into single band TIFFs. Label can stay the same

- [ ] Sentinel-2 13 band TIFF --> 13 single TIFFs for each band

- [ ] Sentinel-1 2 band TIFF --> 2 single band TIFFs for each band

**Comment by echeipesh** [comment]
I assume the the concern is being able to select a subset of bands efficiently. You might get most of the same benefit without changing the file structure by changing from `INTERLEAVE=PIXEL` to `INTERLEAVE=BAND`. The former and current option interleaves pixels with-in the segment, forcing to download all bands in every request. The latter keeps 1 segment = 1 band mapping, allowing readers to select which bands they need to read.

If there are other overriding concerns, please disregard this comment :)

**Comment by tyler-c2s** [comment]
@echeipesh this is correct, mainly for preparing the data to be hosted long term on cloud storage and allowing readers to access only band subsets that they are concerned of (reducing reads). From talking with other users it seemed that splitting bands to separate bands was been preferable, but would be curious if others think `INTERLEAVE=BAND` is a better/comparable solution.

**Comment by tyler-c2s** [comment]
@kbgg would be interested in your thoughts on this as well
